tinygrad/tensor.py

```diff
@@ -136,6 +136,11 @@ def eye(dim, **kwargs): return Tensor([1], **kwargs).slice(((0,dim+1),)).reshape
   @staticmethod
   def arange(stop, start=0, step=1, **kwargs): return Tensor(np.arange(start=start, stop=stop, step=step, dtype=np.float32), **kwargs)
 
+  @staticmethod
+  def where(condition:Tensor, input_:Union[Tensor, float], other:Union[Tensor, float]):
+    c = (condition != 0.0)
+    return c * input_ + (1.0 - c) * other
```

Comment:
not sure whether to have `!= 0` just in case a weird tensor is passed as condition